pyexchange/leverjfutures.py

```diff
@@ -192,6 +191,11 @@ def get_quote_balance(self, quote_asset_address: str) -> str:
         quote_balance = balances[quote_asset_address]['available']
         return quote_balance
 
+    def get_plasma_balance(self, quote_asset_address: str) -> str:
```

Comment:
Might want to use `Address` type instead of `str` as this provides additional safety. Also, it would be good to an `assert(isinstance(quote_asset_address, <TYPE>)` for clearer error messages.